lib/msal-angular/src/msal.service.ts

```diff
@@ -173,11 +173,12 @@ export class MsalService extends UserAgentApplication {
             return null;
         }
 
-        if (this.msalConfig.framework && this.msalConfig.framework.protectedResourceMap) {
+        const frameworkProtectedResourceMap = this.msalConfig.framework && this.msalConfig.framework.protectedResourceMap;
+        if (frameworkProtectedResourceMap) {
             this.getLogger().info("msalConfig.framework.protectedResourceMap is deprecated, use msalAngularConfig.protectedResourceMap");
         }
 
-        const protectedResourceMap = (this.msalConfig.framework && this.msalConfig.framework.protectedResourceMap) || new Map(this.msalAngularConfig.protectedResourceMap);
+        const protectedResourceMap = frameworkProtectedResourceMap && frameworkProtectedResourceMap.size ? frameworkProtectedResourceMap : new Map(this.msalAngularConfig.protectedResourceMap);
```

Comment:
Do we even need to have `protectedResourceMap` in core? Isn't the point to have this only in `Angular` or is it maintained for surface compatibility and non breaking changes?

Reply:
Right, keeping it to avoid breaking changes.

Reply:
okay. Thanks.